program/find-volume-of-a-cuboid/findVolumeOfACuboid.ts

```diff
@@ -0,0 +1,5 @@
+function findVolumeOfACuboid(length: number, width: number, height: number) {
+  return length * width * height;
+}
+
+console.log(2,3,4);
```

Comment:
This should be `console.log(findVolumneOfACuboid(2, 3, 4))`. Otherwise, it looks good

Reply:
Ooh Sorry

Reply:
I've just updated it!

Reply:
No problem and thank you for updating! 👍